src/main/java/net/openhft/chronicle/core/internal/threads/VanillaThreadLock.java

```diff
@@ -1,184 +0,0 @@
-/*
```

Comment:
Should we be deleting the ThreadLock interface as well, VanillaThreadLock was the only implementation and it was introduced with VanillaThreadLock in 4c1b33d49f432c18f54a9954802ed8f22d1acb5a